6_Extending_the_Toolkit-Part2.ipynb

```diff
@@ -8,203 +8,7 @@
     }
```

Comment:
<p>I think we can just mention Azure Pipelines now</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>OK</p>